### DISCUSSION TITLE
Similar one for Bard

### DISCUSSION THREAD

**Discussion by githubdebugger** [matthaigh27/ChatGPT-android-app#22] [General]
I was wondering if you could make similar app/apk for Bard? 

A super lightweight app which we can use on the phone which you described it here https://github.com/matthaigh27/ChatGPT-android-app/discussions/12#discussioncomment-5715457

For sure Google *might* release (a super heavy) one later.

**Comment by matthaigh27** [comment]
Not a bad idea, will have a look!
